pkgs/development/python-modules/ngs-tools/default.nix

```diff
@@ -0,0 +1,44 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchPypi,
+  joblib,
+  numba,
+  numpy,
+  pysam,
+  shortuuid,
+  tqdm,
+  typing-extensions,
+  setuptools-scm,
+}:
+
+buildPythonPackage rec {
+  pname = "ngs-tools";
+  version = "1.8.5";
+  format = "setuptools";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "OA4jahAcWxrDwPzbzJCKIQF5tu8qk/vqn06w7C7cHeA=";
+  };
+
+  propagatedBuildInputs = [
+    joblib
+    numba
+    numpy
+    pysam
+    shortuuid
+    tqdm
+    typing-extensions
+  ];
+
+  nativeBuildInputs = [ setuptools-scm ];
+  pythonImportsCheck = [ "ngs_tools" ];
+
```

Comment:
Could you enable tests with `pytestCheckHook` or similar?

> Make sure the tests are enabled using for example [pytestCheckHook](https://github.com/NixOS/nixpkgs/blob/master/doc/languages-frameworks/python.section.md#using-pytestcheckhook) and, in the case of libraries, are passing for all interpreters. If certain tests fail they can be disabled individually. Try to avoid disabling the tests altogether. In any case, when you disable tests, leave a comment explaining why.

https://github.com/NixOS/nixpkgs/blob/master/doc/languages-frameworks/python.section.md#contributing-guidelines-contributing-guidelines

Reply:
Enabled test using `pytestCheckHook` 
But getting import error

```
==================================== ERRORS ====================================
______________________ ERROR collecting tests/test_bam.py ______________________
ImportError while importing test module '/private/tmp/nix-build-python3.12-ngs-tools-1.8.5.drv-0/ngs-tools-1.8.5/tests/test_bam.py'.
Hint: make sure your test modules/packages have valid Python names.
Traceback:
/nix/store/9pj4rzx5pbynkkxq1srzwjhywmcfxws3-python3-3.12.5/lib/python3.12/importlib/__init__.py:90: in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
tests/test_bam.py:7: in <module>
    from tests.mixins import TestMixin, tqdm_mock
E   ModuleNotFoundError: No module named 'tests'
____________________ ERROR collecting tests/test_binary.py _____________________
ImportError while importing test module '/private/tmp/nix-build-python3.12-ngs-tools-1.8.5.drv-0/ngs-tools-1.8.5/tests/test_binary.py'.
Hint: make sure your test modules/packages have valid Python names.
Traceback:
/nix/store/9pj4rzx5pbynkkxq1srzwjhywmcfxws3-python3-3.12.5/lib/python3.12/importlib/__init__.py:90: in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
tests/test_binary.py:6: in <module>
    from . import mixins
E   ImportError: attempted relative import with no known parent package
___________________ ERROR collecting tests/test_chemistry.py ___________________
ImportError while importing test module '/private/tmp/nix-build-python3.12-ngs-tools-1.8.5.drv-0/ngs-tools-1.8.5/tests/test_chemistry.py'.
Hint: make sure your test modules/packages have valid Python names.
Traceback:
/nix/store/9pj4rzx5pbynkkxq1srzwjhywmcfxws3-python3-3.12.5/lib/python3.12/importlib/__init__.py:90: in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
tests/test_chemistry.py:4: in <module>
    from tests.mixins import TestMixin
E   ModuleNotFoundError: No module named 'tests'
_____________________ ERROR collecting tests/test_fasta.py _____________________
ImportError while importing test module '/private/tmp/nix-build-python3.12-ngs-tools-1.8.5.drv-0/ngs-tools-1.8.5/tests/test_fasta.py'.
Hint: make sure your test modules/packages have valid Python names.
Traceback:
/nix/store/9pj4rzx5pbynkkxq1srzwjhywmcfxws3-python3-3.12.5/lib/python3.12/importlib/__init__.py:90: in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
tests/test_fasta.py:6: in <module>
    from . import mixins
E   ImportError: attempted relative import with no known parent package
_____________________ ERROR collecting tests/test_fastq.py _____________________
ImportError while importing test module '/private/tmp/nix-build-python3.12-ngs-tools-1.8.5.drv-0/ngs-tools-1.8.5/tests/test_fastq.py'.
Hint: make sure your test modules/packages have valid Python names.
Traceback:
/nix/store/9pj4rzx5pbynkkxq1srzwjhywmcfxws3-python3-3.12.5/lib/python3.12/importlib/__init__.py:90: in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
tests/test_fastq.py:8: in <module>
    from . import mixins
E   ImportError: attempted relative import with no known parent package
______________________ ERROR collecting tests/test_gtf.py ______________________
ImportError while importing test module '/private/tmp/nix-build-python3.12-ngs-tools-1.8.5.drv-0/ngs-tools-1.8.5/tests/test_gtf.py'.
Hint: make sure your test modules/packages have valid Python names.
Traceback:
/nix/store/9pj4rzx5pbynkkxq1srzwjhywmcfxws3-python3-3.12.5/lib/python3.12/importlib/__init__.py:90: in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
tests/test_gtf.py:6: in <module>
    from . import mixins
E   ImportError: attempted relative import with no known parent package
___________________ ERROR collecting tests/test_sequence.py ____________________
ImportError while importing test module '/private/tmp/nix-build-python3.12-ngs-tools-1.8.5.drv-0/ngs-tools-1.8.5/tests/test_sequence.py'.
Hint: make sure your test modules/packages have valid Python names.
Traceback:
/nix/store/9pj4rzx5pbynkkxq1srzwjhywmcfxws3-python3-3.12.5/lib/python3.12/importlib/__init__.py:90: in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
tests/test_sequence.py:8: in <module>
    from . import mixins
E   ImportError: attempted relative import with no known parent package
_____________________ ERROR collecting tests/test_utils.py _____________________
ImportError while importing test module '/private/tmp/nix-build-python3.12-ngs-tools-1.8.5.drv-0/ngs-tools-1.8.5/tests/test_utils.py'.
Hint: make sure your test modules/packages have valid Python names.
Traceback:
/nix/store/9pj4rzx5pbynkkxq1srzwjhywmcfxws3-python3-3.12.5/lib/python3.12/importlib/__init__.py:90: in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
tests/test_utils.py:11: in <module>
    from . import mixins
E   ImportError: attempted relative import with no known parent package
=========================== short test summary info ============================
ERROR tests/test_bam.py
ERROR tests/test_binary.py
ERROR tests/test_chemistry.py
ERROR tests/test_fasta.py
ERROR tests/test_fastq.py
ERROR tests/test_gtf.py
ERROR tests/test_sequence.py
ERROR tests/test_utils.py
!!!!!!!!!!!!!!!!!!! Interrupted: 8 errors during collection !!!!!!!!!!!!!!!!!!!!
============================== 8 errors in 0.63s ===============================
error: builder for '/nix/store/ly95bm3jqb625s9qqczr8zc58qdwpvqi-python3.12-ngs-tools-1.8.5.drv' failed with exit code 2;
       last 25 log lines:
       > /nix/store/9pj4rzx5pbynkkxq1srzwjhywmcfxws3-python3-3.12.5/lib/python3.12/importlib/__init__.py:90: in import_module
       >     return _bootstrap._gcd_import(name[level:], package, level)
       > tests/test_sequence.py:8: in <module>
       >     from . import mixins
       > E   ImportError: attempted relative import with no known parent package
       > _____________________ ERROR collecting tests/test_utils.py _____________________
       > ImportError while importing test module '/private/tmp/nix-build-python3.12-ngs-tools-1.8.5.drv-0/ngs-tools-1.8.5/tests/test_utils.py'.
       > Hint: make sure your test modules/packages have valid Python names.
       > Traceback:
       > /nix/store/9pj4rzx5pbynkkxq1srzwjhywmcfxws3-python3-3.12.5/lib/python3.12/importlib/__init__.py:90: in import_module
       >     return _bootstrap._gcd_import(name[level:], package, level)
       > tests/test_utils.py:11: in <module>
       >     from . import mixins
       > E   ImportError: attempted relative import with no known parent package
       > =========================== short test summary info ============================
       > ERROR tests/test_bam.py
       > ERROR tests/test_binary.py
       > ERROR tests/test_chemistry.py
       > ERROR tests/test_fasta.py
       > ERROR tests/test_fastq.py
       > ERROR tests/test_gtf.py
       > ERROR tests/test_sequence.py
       > ERROR tests/test_utils.py
       > !!!!!!!!!!!!!!!!!!! Interrupted: 8 errors during collection !!!!!!!!!!!!!!!!!!!!
       > ============================== 8 errors in 0.63s ===============================
       For full logs, run 'nix-store -l /nix/store/ly95bm3jqb625s9qqczr8zc58qdwpvqi-python3.12-ngs-tools-1.8.5.drv'.
```

While running `pytest` in the ngs-tools, passes all the test
I tried using pytest manually 
```nix
nativeCheckInputs = [ pytest ];
  postInstall = ''
    pytest
  '';
```
But still getting the same error
Can you please tell me what am I missing here?